network/crl.go

```diff
@@ -199,6 +202,10 @@ func (c DefaultCRLClient) Fetch(url string) ([]byte, error) {
 
 		return content, nil
 	case "file":
+		if !allowLocal {
+			return nil, ErrFetchDeniedForLocalURL
+		}
```

Comment:
There should be a test for this.

Reply:
Done, added one small test to ensure `Fetch("file://...", allowLocal = false)` returns `ErrFetchDeniedForLocalURL`